### DISCUSSION TITLE
Differences Between Installing WSL via Microsoft Store vs Using 'wsl --install' Command?

### DISCUSSION THREAD

**Discussion by Uwaang** [microsoft/WSL#10164] [Q&A]
I've recently started using WSL and I've noticed there are a couple of ways to install it: one is via the Microsoft Store and the other is through using the 'wsl --install' command directly in the command prompt.

I've tried searching for information regarding the differences between these two methods, but have been unable to find a clear comparison. Could anyone here kindly shed some light on the key differences? Specifically, I'm curious about potential differences in performance, features, convenience, and updating process.

Any help or clarification would be greatly appreciated. Thanks in advance!

**Comment by Biswa96** [comment]
> 'wsl --install' command directly in the command prompt

That command is just for convenience. Every step, which `wsl --install` does, can be done manually.
